schemas/com.snowplowanalytics.snowplow/browser_context/jsonschema/1-0-0

```diff
@@ -0,0 +1,76 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for browser contexts",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow",
+    "name": "browser_context",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "viewport": {
+      "type": "string",
+      "maxLength": 20,
+      "description": "Viewport dimensions of the browser. Arrives in the form of WidthxHeight e.g. 1200x900."
+    },
+    "documentSize": {
+      "type": "string",
+      "maxLength": 20,
+      "description": "Document dimensions. Arrives in the form of WidthxHeight e.g. 1200x900"
+    },
+    "resolution": {
+      "type": "string",
+      "maxLength": 20,
+      "description": "Device native resolution. Arrives in the form of WidthxHeight e.g. 1200x900"
+    },
+    "colorDepth": {
+      "type": "integer",
+      "minimum": 0,
+      "maximum": 1000,
+      "description": "The number of bits allocated to colors for a pixel in the output device, excluding the alpha channel."
+    },
+    "devicePixelRatio": {
+      "type": ["integer", "null"],
+      "minimum": 0,
+      "maximum": 1000,
```

Comment:
Nit: the `maximum` and `maxLength` values in most of these seem quite loose. If we don't know the bounds or are not sure then lets keep it like this, but in case there is a known limit that we could set, I think there is a potential it could help filter out some fabricated or bot events in some cases if we validate the values.

Reply:
The limits are kind of arbitrary with some padding on future values since I thought we could be a bit more lenient.
 _(Probably a DPR cannot reach 1000 soon 😆 )_

>  I think there is a potential it could help filter out some fabricated or bot events in some cases if we validate the values

So in that one you mean by setting a standard limit and let them fall through to the bad rows ?

Reply:
Right, that's what I meant – for the events with obviously weird values to go to bad rows. But it's a nitpick that has pretty marginal value so if it's difficult to say for sure, just leave the lenient setting.